docs/guide/search/elastic.md

```diff
@@ -0,0 +1,573 @@
+
+    Number of retries
+
+    With this setting you can configure the number of attempts that eZ platform makes to connect
+    to the nodes of the cluster before it throws an exception.
+    By default, the `null` setting is used, which means that the number of retries equals to the number of nodes in the cluster.
+
+    ``` yaml
+    <connection_name>:
+      # ...
+      retries: null
+    ```
+
+    Depending on the connection pool that you select, eZ Platform's reaction to reaching the maximum number of retries might differ.
+
+    For more information, see [Set retries](https://www.elastic.co/guide/en/elasticsearch/client/php-api/7.x/configuration.html#_set_retries).
+
+    ##### Configure Elasticsearch Cloud
+
+    As an alternative, you might decide to use Elasticsearch Cloud, a commercial SaaS solution.
+    With Elasticsearch Cloud you do not have to build or manage your own Elasticsearch cluster.
+    Also, you do all the configuration and administration in a graphical user interface.
+
+    To connect to a cloud solution with eZ Platform, you must set the `elastic_cloud_id` parameter by
+    providing an alphanumerical ID string that you obtain from the cloud's user interface, for example:
+
+    ``` yaml
+    <connection_name>:
+      elastic_cloud_id: 'production:ZWFzdHVzMi5henVyZS5lbGFzdGljLWNsb3VkLmNvbTo5MjQzJGUwZ'
+    ```
+
+    With the ID set, you must configure authentication to be able to access the remote environment.
+
+    ##### Configure security
+
+    Elasticsearch instances support `basic` and `api_key` authentication methods.
+    You select authentication type and configure the settings under the `authentication` key. By default, authentication is disabled:
+
+    ``` yaml
+    <connection_name>:
+      # ...
+    	authentication:
+    	  type: null
+    ```
+
+    If you connect to Elasticsearch hosts outside of your local network, you might also need to configure SSL encryption.
+
+    #### Configure basic authentication
+
+    If your Elasticsearch server is protected by HTTP authentication, you must provide eZ Platform with the credentials, so that eZ Platform requests can be authenticated by the server.
+    Basic authentication support requires that you pass a set of parameters that is similar to the following example:
+
+    ``` yaml
+    <connection_name>
+      # ...
+    	  authentication:
+    		  type: basic
+    		  credentials: ['<user_name', '<password>']
+    ```
+
+    For example:
+
+    ``` yaml
+    ezplatform_elastic_search_engine:
+      connections:
+        cloud:
+          debug: true
+          elastic_cloud_id: 	'test:ZWFzdHVzMi5henVyZS5lbGFzdGljLWNsb3VkLmNvbTo5MjQzJGUwZ'
+          authentication:
+    	      type: basic
+    	      credentials: ['elastic', '1htFY83VvX2JRDw88MOkOejk']
+    ```
+
+    #### Configure API key authentication
+
+    If your Elasticsearch cluster is protected by API keys, you must use the values described below to
```

Comment:
```suggestion
    If your Elasticsearch cluster is protected by API keys, you must provide the key and secret in authentication configuration to
```
Something like this - IMHO there's too much of a jump between "described below" and the actual example.